demos/gaze_estimation_demo/cpp_gapi/include/head_pose_estimator.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstdio>
+#include <string>
+
```

Comment:
```suggestion
```
same for others

Reply:
File is removed.

Reply:
There are others. At least `exponential_averager.hpp` having these includes.

Reply:
Removed extra `include`s.